appcues/src/main/java/com/appcues/data/mapper/experience/ExperienceMapper.kt

```diff
@@ -93,6 +93,12 @@ internal class ExperienceMapper(
 
     private fun List<ExperimentResponse>.getExperiment(experienceId: UUID) =
         this.firstOrNull { it.experienceId == experienceId }?.let { experimentResponse ->
-            Experiment(experimentResponse.experimentId, experimentResponse.group)
+            Experiment(
+                id = experimentResponse.experimentId,
+                group = experimentResponse.group,
+                experienceId = experimentResponse.experienceId,
+                goalId = experimentResponse.goalId,
+                contentType = experimentResponse.contentType
```

Comment:
what is contentType?

Reply:
I don't really know. it was a property the backend requested for analytics purposes though. I can try to find out.